apps/hono/.env.example

```
@@ -0,0 +1,66 @@
+# Opaque secret used to sign and verify JWT tokens
+
+# openssl rand -base64 32
+AURA_AUTH_SALT=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)